server/server.ts

```diff
@@ -1,10 +1,13 @@
 /* eslint-disable */
 require('dotenv').config();
+// @ts-ignore
```

Comment:
What were the errors you were getting?

Reply:
These kind of errors. I plan to fix them when I convert everything to TS.

```
tests/knexfile-generator.test.ts:10:5 - error TS2451: Cannot redeclare block-scoped variable 'NODE_ENV'.

10     NODE_ENV,
       ~~~~~~~~

  server/server.ts:3:9
    3 const { NODE_ENV, PORT, ENABLE_SQREEN } = process.env;
              ~~~~~~~~
    'NODE_ENV' was also declared here.
```

Reply:
Don't love merging them in but we want to get 1.0 out ASAP so I guess it's not blocking.

Reply:
Exactly my thoughts